### ISSUE TITLE
Migrate plugins to use ui plugin

### ISSUE THREAD

**Issue by victorzinho** [geoladris/plugins#14]
In [feature_ui_migration](https://github.com/geoladris/plugins/tree/feature_ui_migration) branch.

- [x] auth
- [x] base
- [x] feedback
- [x] footnote
- [x] geoexplorer-reader
- [x] language-buttons
- [x] layer-order
- [x] layers-editor
- [x] layer-time-sliders
- [x] time-slider
- [x] tour

**Comment by victorzinho** [comment]
Some progress has been done. We are currently blocked with:

* `base`: It requires [proper sliders](https://github.com/csgis/geoladris-plugins/issues/7) (for `active-layer-list.js`),  [creating elements without a parent](https://github.com/csgis/geoladris-plugins/issues/6) (for `layer-info-button.js`), [adding elements to body](https://github.com/csgis/geoladris-plugins/issues/6) (for `layout.js` and `layer-info-button.js`) and uses `fancybox` (for `show-info.js`).
* `layer-order`: It uses `sortable` from jquery-ui; we need to decide what to do here.
* `layers-editor`: It requires [creating elements without a parent](https://github.com/csgis/geoladris-plugins/issues/6). It uses `sortable` from jquery-ui; we need to decide what to do here.
* `layer-time-sliders`: It requires [proper sliders](https://github.com/csgis/geoladris-plugins/issues/7).
* `time-slider`: It requires [proper sliders](https://github.com/csgis/geoladris-plugins/issues/7) and [changing a input's label](https://github.com/csgis/geoladris-plugins/issues/8).
* `tour`: uses `tipsy`; we need to decide what to do here.

It is possible to continue with:

* `base/layer-list.js` (partially migrated).
* `base/info-dialog.js`
* `base/layer-list-selector.js`

**Comment by victorzinho** [comment]
All [geoladris-plugins](https://github.com/csgis/geoladris-plugins/milestone/1) issues have been fixed in a [refactoring_ui](https://github.com/csgis/geoladris-plugins/tree/refactor_ui) branch (not closed because they have not been merged into master yet). 

`tipsy` and `sortable` functionalities have been included in the ui plugin. We should be able to replace `fancybox` with `ui.create("dialog", {...})`.

Everything should be unblocked now. The list of migrated plugins ([first comment #1](https://github.com/geoladris/plugins/issues/14#issue-202131875)) is updated. Specifically, in base:

- [x] active-layer-list.js
- [x] banner.js
- [x] info-dialog.js
- [x] layer-info-button.js
- [x] layer-list.js
- [x] layer-list-selector.js
- [x] layout.js
- [x] legend-button.js
- [x] legend-panel.js
- [x] loading.js
- [x] show-info.js
- [x] toolbar.js
- [x] zoom-bar.js
